src/neo/Utility.cs

```diff
@@ -17,6 +17,17 @@ public Logger()
             }
         }
 
+        static Utility()
+        {
+            // Unify unhandled exceptions
+            AppDomain.CurrentDomain.UnhandledException += CurrentDomain_UnhandledException;
```

Comment:
Exceptions that occur before the `Utility` class is used will not be handled. I suggest listening to the `UnhandledException` event in `NeoSystem`'s static constructor.